step-plans/step-plans-base-artefacts/src/main/java/step/artefacts/AfterSequence.java

```diff
@@ -18,8 +18,12 @@
  ******************************************************************************/
 package step.artefacts;
 
+import step.automation.packages.AutomationPackageNamedEntity;
 import step.core.artefacts.Artefact;
+import step.plans.parser.yaml.model.YamlModel;
 
+@YamlModel
+@AutomationPackageNamedEntity(name = "afterSequence")

```

Comment:
Can we have a AutomationPackageNamedEntity which is not a YamlModel? i.e. do we need both annotations?

Reply:
@iegorov777 
Agree. I think we can only the `YamlModel` annotation (as more general) and add the the new `yamlName` propery to   `YamlModel` to define the name.

Reply:
@iegorov777 agreed

Reply:
@david-stephan 
Implemented